src/xcode/ENA/ENA/Source/Workers/SQLiteKeyValueStore.swift

```diff
@@ -45,15 +46,16 @@ class SQLiteKeyValueStore {
 		}
 	}
 
+	/// - returns: `Data` if the key/value pair in the DB, `nil` otherwise
 	private func getData(for key: String) -> Data? {
 		openDbIfNeeded()
 
 		do {
 			let query = "SELECT value FROM kv WHERE key = ?;"
 			let result = try db.executeQuery(query, values: [key])
-			var resultData = Data()
+			var resultData: Data?
 			while result.next() {
-				guard let data = result.data(forColumn: "value") else {
+				guard let data = result.dataNoCopy(forColumn: "value") else {
```

Comment:
I'd like to call out attention to this change here - everything seems to work fine but this change had to be made to support getting empty `Data` out of the DB. Without this, behavior is as follows.

Say we do this `kvStore["hello"] = Data()`, we should arguable expect `kvStore["hello"]?.isEmpty == true`
However with the current code this is not the behavior, we get this:
`result.columnIsNull("value")` -> `false`
`result.data(forColumn: "value")` -> `nil` but if we do instead
`result.dataNoCopy(forColumn: "value")` -> empty `Data`

To me this seemed the easiest fix, but it doesn't quite sit right - even though the tests pass.